neo4j_utils/_driver.py

```diff
@@ -1603,3 +1603,17 @@ def last_fail(self):
         """
 
         return self._queries.get('last_failed')
+
+    def get_neo4j_version(self):
```

Comment:
maybe remove the `get_` prefix and make it a `@property`

Reply:
What would be the advantage of having a property instead?